UnitsNet.Tests/IValueQuantityTests.cs

```diff
@@ -50,5 +50,33 @@ public void IValueQuantityTDecimal_AsUnitSystem_ReturnsDecimal()
             IValueQuantity<decimal> decimalQuantity = Power.FromMegawatts(1234.5);
             Assert.IsType<decimal>(decimalQuantity.As(UnitSystem.SI));
         }
+
+        [Fact]
+        public void IValueQuantityTDouble_ToUnitEnum_ReturnsIValueQuantityTDouble()
+        {
+            IValueQuantity<double> doubleQuantity = Temperature.FromDegreesCelsius(1234.5);
+            Assert.IsAssignableFrom<IValueQuantity<double>>(doubleQuantity.ToUnit(TemperatureUnit.Kelvin));
+        }
+
+        [Fact]
+        public void IValueQuantityTDouble_ToUnitUnitSystem_ReturnsIValueQuantityTDouble()
+        {
+            IValueQuantity<double> doubleQuantity = Temperature.FromDegreesCelsius(1234.5);
+            Assert.IsAssignableFrom<IValueQuantity<double>>(doubleQuantity.ToUnit(UnitSystem.SI));
+        }
+
+        [Fact]
+        public void IValueQuantityTDecimal_ToUnitEnum_ReturnsIValueQuantityTDecimal()
+        {
+            IValueQuantity<decimal> decimalQuantity = Power.FromWatts(1234.5);
```

Comment:
Use Information instead, since Power will be changed to double in v6.

Reply:
@angularsen that will work for the Enum overload, but Information doesn't support SI units.

Reply:
Just noticed we do the same for ```IValueQuantityTDecimal_AsUnitSystem_ReturnsDecimal```.